src/components/dashboard/Dashboard.js

```diff
@@ -293,9 +303,10 @@ const Dashboard = props => {
 
     log.debug('initDashboard subscribed to feed')
     handleDeleteRedirect()
-    handleResize()
     animateClaim()
     InteractionManager.runAfterInteractions(handleAppLinks)
+
+    Dimensions.addEventListener('change', handleResize)
```

Comment:
initDashboard is called in after component mounted effect. as we have now event subscriptions here, we have cancel them on unmount
1. add disposeDashboard function which removes event listeners from Dimensions (and probably other listneners we have / other stuff should be cancelled/removed)
2. call in in the cancel hook